CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # CHANGELOG
 
+### 1.4.0 (Unreleased)
```

Comment:
please don't edit the changelog in Prs. The release manage (me) will update the changelog when we do the release.